aws_lambda_powertools/utilities/data_classes/rabbit_mq_event.py

```diff
@@ -83,13 +82,13 @@ def data(self) -> str:
     @property
     def decoded_data(self) -> str:
         """Decodes the data as a str"""
-        return base64.b64decode(self.data.encode()).decode()
+        return base64_decode(self.data)
 
     @property
     def json_data(self) -> Any:
```

Comment:
note to future (not this PR): we should have a class decorator to handle these ubiquitous JSON and Base64 properties